packages/ui/v2/core/Button.tsx

```diff
@@ -43,13 +43,16 @@ const variantClassName = {
   secondary: "border border-gray-200 text-brand-900 bg-white hover:bg-gray-100",
   minimal:
     "text-gray-700 bg-transparent hover:bg-gray-100 focus:outline-none focus:ring-2 focus:ring-offset-1 focus:bg-gray-100 focus:ring-brand-900 dark:text-darkgray-900 hover:dark:text-gray-50",
+  minimalSecondary:
```

Comment:
Created a new variant for minimal + outline hover - as we have a few designs that use both minimal/minimalSecondary